airflow/providers/snowflake/operators/s3_to_snowflake.py

```diff
@@ -51,7 +51,7 @@ def __init__(self,
                  table,
                  stage,
                  file_format,
-                 schema,
+                 schema,  # TODO: shouldn't be required
```

Comment:
Can you tell more about it?

Reply:
Similar to database/warehouse/role values, the schema may already be set from the user's settings (`DEFAULT_NAMESPACE`) or in the session [set via connection extras](https://github.com/apache/airflow/blob/7d4b81ddd7f0c11532b0abac2050336b4f243b88/airflow/providers/snowflake/hooks/snowflake.py#L62). It can also be explicitly referenced in the table name, i.e. `my_schema.my_table`. Although there are good reasons to override the session default, I think that should be optional and is consistent with [SnowflakeOperator](https://github.com/apache/airflow/blob/7d4b81ddd7f0c11532b0abac2050336b4f243b88/airflow/providers/snowflake/operators/snowflake.py#L44-L46).

Will update comment to be a little more specific.